windows/src/global/delphi/chromium/Keyman.System.CEFManager.pas

```diff
@@ -116,7 +113,9 @@ constructor TCEFManager.Create;
 
   GlobalCEFApp.OnProcessMessageReceived := GlobalCEFApp_ProcessMessageReceived;
 
-  TKeymanSentryClient.OnBeforeShutdown := Self.Cleanup;
+  // We no longer attempt to cleanup before shutdown, and rely on the child cef
+  // processes to do the cleanup, which is more reliable.
+  // TKeymanSentryClient.OnBeforeShutdown := Self.Cleanup;
```

Comment:
... does `Self.Cleanup` get used at all anymore?  I tried searching through the changes, but I don't see any references to it among this PR's added lines.

Reply:
`Self.Cleanup` is used by the class destructor.